### PR TITLE
Delete Reason + Minor Change

### PR DESCRIPTION
**Pull request by TheGamer3514**

### CODE REVIEW

**Comment by TejasLamba2006** [comment]
Isnt this some useless changes that are not required?

**Comment by TheGamer3514** [comment]
no, it adds a delete reason. something ishaan wanted to add

File: src/pages/botlist/delete.ejs [hunk 1]
```diff
@@ -1,33 +1,30 @@
...
+<br><br><br><br><br>
+<div class="container">
+  <center>
+    <h2 class="title"><b><i class="fa-solid fa-trash"></i> Delete Bot</b></h2>
+    <p>Fill out the reason below in order to delete the bot.</p>
+  </center>
+  <form action="/bots/<%- id %>/delete" method="POST">
+    <div class="form-group required">
+      <label for="reason">Reason</label>
+      <input type="text" class="form-control" id="reason" name="reason" placeholder="This bot is stupid." autocomplete="off">
```
Comment (TejasLamba2006) [inline src/pages/botlist/delete.ejs:22]: You are taking a reason here, But where are the shit you sending it?

You havnt done anything in the backend or adding the reason to the database or sending it in discord.

How will ishaan see it?

**Review by TejasLamba2006 — CHANGES_REQUESTED** [review]

**Comment by TheGamer3514** [comment]
i did modify the backend?

**Comment by TejasLamba2006** [comment]
Please send it I coudnt locate it

**Pull request merged**